qrcode/idf_component.yml

```diff
@@ -1,5 +1,5 @@
-version: "0.1.0~1"
+version: "0.1.0~2"
 description: QR Code generator
 url: https://github.com/espressif/idf-extra-components/tree/master/qrcode
 dependencies:
-  idf: ">=4.3"
+  idf: ">=4.0"
```

Comment:
Do we even need this under dependencies?

Reply:
@shahpiyushv makes sense, as component manager anyway is not supported for `IDF <= 4.0`! Made the change.